pdf_parser.py

```diff
@@ -182,25 +174,31 @@ def populate_charges_record_table(self, parser_obj):
                 #value = parser_obj.horizontal_table[index]
                 records_id = 0
                 length_fields = len(parser_obj.horizontal_table[index])
-                while(length_fields == 4):
+                while(len(parser_obj.horizontal_table[index]) == 4):
```

Comment:
@bhavaniab by taking length_fields the loop is stopping in between and as a result it is not able to fetch all the results